src/base1/cockpit.css

```diff
@@ -313,15 +313,27 @@ a {
 }
 
 /* Our terminal or logs */
+.console-ct-container {
+    border: none;
+}
+
 .console-ct {
     font-family: Menlo, Monaco, Consolas, monospace;
-    margin-top: 0;
-    margin-bottom: 0;
+    background-color: black;
```

Comment:
console-ct is used by both terminals and log output displays. See the screenshots of the kubernetes and docker logs attached to this bug. I don't think this should have a black background.